spec/models/book_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe Book do
+  subject(:book) { build(:book) }
+
+  it do
+    should validate_presence_of(:genre)
```

Comment:
Do it in one line `  it { should validate_presence_of(:genre) }`

Reply:
These are ruby blocks.
Here you have some examples:
https://mixandgo.com/learn/mastering-ruby-blocks-in-less-than-5-minutes